api/dto/triggers.go

```diff
@@ -377,30 +377,31 @@ type TriggerMaintenance struct {
 	Metrics map[string]int64 `json:"metrics"`
 }
 
-func (*TriggerMaintenance) Bind(r *http.Request) error {
+func (*TriggerMaintenance) Bind(*http.Request) error {
 	return nil
 }
 
 type ThrottlingResponse struct {
 	Throttling int64 `json:"throttling"`
 }
 
-func (*ThrottlingResponse) Render(w http.ResponseWriter, r *http.Request) error {
+func (*ThrottlingResponse) Render(http.ResponseWriter, *http.Request) error {
 	return nil
 }
 
 type SaveTriggerResponse struct {
-	ID      string `json:"id"`
-	Message string `json:"message"`
+	ID      string               `json:"id"`
+	Message string               `json:"message"`
+	Check   TriggerCheckResponse `json:"check,omitempty"`
```

Comment:
here will be errors, yea?

Reply:
Yes, it is response struct for create/update APIs. Now it contains result of validation with tree of problems, and each problem has type (yellow/red). So `Check` field is result of existing /check/ API

Reply:
not sure with naming, maybe as in "check" named "check_result", but nothing is in my mind better 
maybe Oleg suggests smth

Reply:
`check_result` seems fine. Simple `check` seems a little to general

Reply:
Well, I will rename to `checkResult` (JSON is camelCase)

Reply:
Renamed field and JSON key, https://github.com/moira-alert/moira/pull/817/commits/10f6c6e91319b1b768ddf8c53cb9242cc7283f94